src/plugins/intel_cpu/src/utils/blob_dump.cpp

```diff
@@ -102,6 +102,13 @@ void BlobDumper::prepare_plain_data(const MemoryPtr &memory, std::vector<uint8_t
     const void *ptr = memory->getData();
 
     switch (desc.getPrecision()) {
+        case Precision::FP16: {
```

Comment:
minor: better to put right after (or before) BF16 to keep precision order by size.